.gitignore

```diff
@@ -42,3 +42,6 @@ yarn.lock
 
 /docs/docs
 /docs/docs/
+
+# CODEOWNERS
+/.github/CODEOWNERS
```

Comment:
@alexisintech why do we want CODEOWNERS in our `.gitignore`?

Reply:
tbh i have no clue why i did that today